### DISCUSSION TITLE
Making pipe-viewer compatible with the mpv sponsorblock script.

### DISCUSSION THREAD

**Discussion by canisllup** [trizen/pipe-viewer#178] [General]
So, there is this script https://github.com/po5/mpv_sponsorblock which works really well when you use `mpv <youtube_link>` or with ytfzf, but not with pipe-viewer. I think the reason is because of the name of the `file` that pipe-viewer grabs and it's not recognized by the sponsorblock script. Any ideas how we could make this work? Either on pipe-viewers side or on the script? Of course one way to _fix_ this would be by using youtube-viewer, but I'd rather stick with pipe.

**Comment by trizen** [comment]
See: https://github.com/trizen/pipe-viewer/issues/13
